source/_components/openuv.markdown

```diff
@@ -0,0 +1,156 @@
+`scan_interval` key), but please note that over-running the API will require
+you to upgrade to a paid plan (and may disable your access in the meantime).
+</p>
+
+## {% linkable_title Base Configuration %}
+
+To retrieve data from OpenUV, add the following to your `configuration.yaml`
+file:
+
+```yaml
+openuv:
+  api_key: YOUR_OPENUV_API_KEY
+```
+
+To configure additional functionality, add configuration options beneath
+a `binary_sensor` and/or `sensor` key within the `openuv` section of
+`configuration.yaml` as below:
+
+```yaml
+openuv:
+  api_key: YOUR_OPENUV_API_KEY
+  binary_sensors:
+    # binary sensor configuration options...
+  sensors:
+    # sensor configuration options...
+```
+
+{% configuration %}
+api_key:
+  description: your OpenUV API key
+  required: true
+  type: string
+scan_interval:
+  description: the frequency (in seconds) between data updates
+  required: false
+  type: int
+  default: 1800
+binary_sensors:
+  description: binary sensor-related configuration options
+  required: false
+  type: map
+  keys:
+    monitored_conditions:
+      description: the conditions to create sensors from
+      required: false
+      type: list
+      default: all (`uv_protection_window`)
+sensors:
+  description: sensor-related configuration options
+  required: false
+  type: map
+  keys:
+    monitored_conditions:
+      description: the conditions to create sensors from
+      required: false
+      type: list
+      default: all ( `current_ozone_index`, `current_uv_index`, `max_uv_index`, `safe_exposure_time_type_1`, `safe_exposure_time_type_2`, `safe_exposure_time_type_3`, `safe_exposure_time_type_4`, `safe_exposure_time_type_5`, `safe_exposure_time_type_6` )
+{% endconfiguration %}
+
+## {% linkable_title Binary Sensor Types %}
+
+### {% linkable_title `uv_protection_window` %}
+
+This sensor displays `true` if UV protection (sunscreen, etc.) is recommended
+at the current date and time; displays `false` otherwise.
+
+## {% linkable_title Sensor Types %}
+
+### {% linkable_title `current_ozone_level` %}
+
+This sensor displays the current ozone level in du (Dobson Units).
+
+### {% linkable_title `current_uv_index` %}
+
+This sensor displays the current UV index.
+
+### {% linkable_title `max_uv_index` %}
+
+This sensor displays the maximum UV index that will be encountered that day (at
+solar noon).
+
+### {% linkable_title `safe_exposure_time_type_X` %}
+
+These sensors (up to six in quantity, depending on which conditions are
+monitored) display the approximate number of minutes a particular skin type
+can be exposed to the sun before burning/tanning starts. Each sensor depicts
+a skin type based on the
+[Fitzpatrick scale](https://en.wikipedia.org/wiki/Fitzpatrick_scale):
+
+#### {% linkable_title Skin Type I %}
+
+* Typical Features: Very fair skin, white; red or blond hair; light-colored eyes; freckles likely
+* Tanning Ability: Always burns, does not tan
+* Ethnicity: Scandinavian, Celtic
```

Comment:
These ethnicities don't cover everything, so I would prefer to instead link to the source.

Maybe remove all the characteristics and just list the skin types 1-6 and refer to the wiki source for more info?